walnut/cmd/convert.go

```diff
@@ -58,6 +63,14 @@ specify which modules should be used when generating task templates. Control-OCC
 
 	Run: func(cmd *cobra.Command, args []string) {
 		for _, dumpFile := range args {
+			// Strip .json from end of filename
+			nameOfDump := dumpFile[:len(dumpFile)-5]
+			defaults := map[string]string {
+				"user": "flp",
+				nameOfDump + "_monitoring_qc_url": "no-op://",
+				"dpl_config": "",
```

Comment:
Left empty until we figure out a way to provide the location of the DPL dump.

Reply:
Agreed. Please ensure a JIRA ticket exists for this specific issue.

Reply:
I believe this should be sufficient: https://alice.its.cern.ch/jira/browse/OCTRL-274